Sources/Screens/RepositoryList/RepositoryListView.swift

```diff
@@ -41,11 +41,13 @@ struct RepositoryListView: View {
             }
         }
         .sheet(item: $viewModel.destination, onDismiss: {
-            viewModel.destination = nil
+//            viewModel.destination = nil
```

Comment:
If something is not used anymore, instead of commenting it, you can remove it from the source control. :)